motion.service.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Motion daemon
+After=local-fs.target
```

Comment:
I know absolutely nothing about systemd etc - but I wonder if we should be waiting until the network is available as well as the fs? Not sure how motion will react if it's started before network interfaces are brought up.